SpinnerDatePickerLib/src/main/java/com/tsongkha/spinnerdatepicker/DatePickerDialog.java

```diff
@@ -44,10 +45,23 @@
         void onDateSet(DatePicker view, int year, int monthOfYear, int dayOfMonth);
     }
 
+    /**
+     * Callback for when things are cancelled
+     */
+    public interface OnDateCancelListener {
+        /**
+         * Called when cancel happens.
+         *
+         * @param view The view associated with this listener.
+         */
+        void onCancelled(DatePicker view);
```

Comment:
Hi! thanks for the pull request. Just wondering what the use case is for passing the view inside the callback?